tradeexecutor/ethereum/uniswap_v3_routing.py

```diff
@@ -103,7 +105,7 @@ def trade_on_router_three_way(self,
         if check_balances:
             self.check_has_enough_tokens(quote_token, reserve_amount)
 
-        pool_fees = [intermediary_pair.fee, target_pair.fee]
+        pool_fees = [int(intermediary_pair.fee), int(target_pair.fee)]
```

Comment:
Can you add a comment here when this is 0..1, when BPS, when something else just for claification

Reply:
Maybe add a safety asset that the fee range looks correct... because I run to some of these mix ups earlier

Reply:
The source of the integer error was coming from `translate_trading_pair`, which got fixed in a different branch. 

https://github.com/tradingstrategy-ai/trade-executor/blob/c67fbeae285ff429514547ffa2f49acc25663c06/tradeexecutor/strategy/trading_strategy_universe.py#L879-L880